### PR TITLE
weight fix

### PR DESCRIPTION
**Pull request by LiMoon**
19033 weight : 20
22826 weight : 1

### CODE REVIEW

**Comment by Lemongrass3110** [comment]
Hey @LiMoon.
Could you please associate your gmail address with your github account?
Thank you!

**Comment by LiMoon** [comment]
@Lemongrass3110 public email ?
my primary email address is private.

**Comment by Lemongrass3110** [comment]
I think you noticed it already, because your commit 14fe482 is now associated correctly with your account.

**Comment by secretdataz** [comment]
I can't find any informations about your newly added item. Please provide sources and correct your changes accordingly.

File: db/re/item_db.txt [hunk 1]
```diff
@@ -4004,6 +4004,7 @@
 5919,Camellia_Hair_Pin,Camellia Hair Pin,4,0,,1000,,,,1,0xFFFFFFFF,63,2,256,1,50,1,1203,{ bonus bMdef,20; bonus3 bAutoSpell,"AL_PNEUMA",1,30+getrefine()*5; /* CONFIRM The Rate*/ },{},{}
 5920,Medical_Boots,Medical Boots,4,0,,300,,10,,0,0xFFFFFFFF,63,2,64,,10,1,,{ .@r = getrefine(); bonus bHealPower,10+((.@r/2) * 2); bonus2 bSkillUseSP,"AB_CHEAL",(.@r * 5); },{},{}
 5943,Owl_viscount_Silk_Hat,Owl Viscount Silk Hat,4,0,,500,,15,,1,0xFFFFFFFF,63,2,256,1,70,1,1323,{ bonus bInt,1; bonus bAspdRate,10; .@r = getrefine(); bonus bMatk,7*.@r; autobonus "{ .@r = getrefine(); bonus bAspdRate,(.@r >= 9 ? 40 : ((.@r >= 7) ? 10 : ((.@r >= 5) ? 5 : 1))); }",.@r*20,30000,BF_MAGIC,"{ specialeffect2 EF_SPELLBREAKER }"; },{},{}
+5946,Drama_Clockwork,Drama Clockwork,4,0,,300,,5,,1,0xFFFFFFFF,63,2,256,1,20,1,1335,{},{},{}
```
Comment (secretdataz) [inline db/re/item_db.txt]: Incomplete script.

File: db/re/item_db.txt [hunk 2]
```diff
@@ -8527,6 +8528,7 @@
 14765,Limited_Edition_JOB_Battle_Manual,Limited Edition JOB Battle Manual,18,10,,10,,,,0,0xFFFFFFFF,63,2,,,,,,{ sc_start SC_JEXPBOOST,3600000,35; },{},{}
 14766,Limited_Power_Booster,Limited Power Booster,18,10,,10,,,,0,0xFFFFFFFF,63,2,,,,,,{ bonus_script "{ bonus bBaseAtk,30; bonus bMatk,30; bonus bAtkRate,1; bonus bMatkRate,1; bonus bHit,30; bonus bFlee,30; bonus bAspd,1; bonus bUseSPrate,-5; bonus bFixedCastrate,-30; }",1800,1,1; },{},{}
 14805,Almighty_Lucky_Egg,Almighty Lucky Egg,18,0,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getgroupitem(IG_Almighty_Lucky_Egg); },{},{}
+
```
Comment (secretdataz) [inline db/re/item_db.txt]: Extra whitespace?

File: db/re/item_db.txt [hunk 3]
```diff
@@ -9846,6 +9848,8 @@
 19157,Drooping_Rebellion_,Drooping Rebellion,4,20,,200,,1,,1,0xFFFFFFFF,63,2,256,,1,1,1444,{ bonus bAllStats,1; bonus bUseSPrate,-10; .@r = getrefine(); if (.@r >= 9) { bonus bAspd,1; } if (.@r >= 12) { bonus bSpeedRate,25; } bonus2 bExpAddRace,RC_All,5; bonus2 bDropAddRace,RC_All,5; },{},{}
 19158,C_GeminiS58_Eyes_Blue,Costume GeminiS58 Eyes Blue,4,0,,0,,0,,0,0xFFFFFFFF,63,2,2048,,0,0,1456,{},{},{}
 19159,Barrel_Helm,Barrel Helm,4,20,,420,,59,,1,0xFFFFFFFF,63,2,769,,60,1,1498,{ bonus bVit,5; bonus bHPrecovRate,-10; .@r = getrefine(); if (.@r >= 7) { bonus bDef,100; skill "CR_AUTOGUARD",1; } if (.@r >= 8) { bonus2 bSubRace,RC_DemiHuman,10; bonus2 bSubRace,RC_Player,10; if(BaseJob==Job_SuperNovice) { skill "CR_AUTOGUARD",5; } } },{},{}
+19160,Male_Poring_Balloon,Male Poring Balloon,4,20,,50,,0,,1,0xFFFFFFFF,63,2,1,,1,1,1159,{},{},{}
```
Comment (secretdataz) [inline db/re/item_db.txt]: Please provide sources.

File: db/re/item_db.txt [hunk 4]
```diff
@@ -10723,6 +10727,7 @@
 20762,C_Great_Devil_Wings,Costume Great Devil Wings,4,0,,0,,0,,0,0xFFFFFFFF,63,2,8192,,1,0,10,{},{},{}
 20763,C_Amistr_Bag,Costume Amistr Bag,4,0,,0,,0,,0,0xFFFFFFFF,63,2,8192,,1,0,4,{},{},{}
 20764,C_Fallen_Angel_Wing,Costume Fallen Angel Wing,4,0,,0,,0,,0,0xFFFFFFFF,63,2,8192,,1,0,3,{},{},{}
+//20774,,,4,20,,500,,8,,1,0xFFFFFFFF,63,2,4,,1,1,0,{},{},{}
```
Comment (secretdataz) [inline db/re/item_db.txt]: Incomplete entry

**Comment by secretdataz** [comment]
Merged in 71c8cc3. Thank you.

**Pull request closed without merging**